docsite/rst/playbooks_lookups.rst

```diff
@@ -347,6 +347,96 @@ TLSA         usage, selector, mtype, cert
 TXT          strings
 ==========   =============================================================================
 
+.. _mongodb_lookup:
+
+MongoDB Lookup
+``````````````
+.. versionadded:: 2.1.1
+
+.. warning:: This lookup depends on the `pymongo <http://www.mongodb.org/>`_
+             library.
+
+
+The ``MongoDB`` lookup runs the *find()* command on a given *collection* on a given *MongoDB* server.
+
+The result is a list of jsons, so slightly different from what PyMongo returns. In particular, *timestamps* are converted to epoch integers.
+
+Currently, the following parameters are supported.
+
+======================  =========   =======   ====================   =======================================================================================================================================================================
+Parameter               Mandatory   Type      Default Value          Comment
+----------------------  ---------   -------   --------------------   -----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+connection_string       no          string    mongodb://localhost/   Can be any valid MongoDB connection string, supporting authentication, replicasets, etc. More info at https://docs.mongodb.org/manual/reference/connection-string/
```

Comment:
it would be harder to parse, increase the complexity of the source code and the amount of test cases.

the DB must be given separately anyway on some corner cases. so this is just the generic case.